
syntax="proto3";

package User_Proto;

option cc_generic_services=true;

message ErrorCode
{
    int32 errcode=1;
    bytes errmsg=2;
}
message LoginRequest
{
    bytes name=1;
    bytes pwd=2;
}
message LoginResponse
{
    ErrorCode result=1;
    bool success=2;
}
service UserServiceRpc
{
    rpc Login(LoginRequest) returns(LoginResponse);
}


